pkg/placement/controller.go

```diff
@@ -112,18 +115,21 @@ func NewController(mgr ctrlm.Manager, wdsRestConfig *rest.Config, imbsRestConfig
 
```

Comment:
another comment that is probably leftover from Paolo's code.
receiving in the NewController args rest.config is not testable. 
clients should be created in the cmd and received as args in pkg.
this way NewController should get arguments from types: `dynamic.Interface`,`kubernetes.Interface`,`apiextensionclientset.Interface` which will allow to test this code with fake implementations